.ci/scripts/gather_test_models.py

```diff
@@ -72,7 +72,7 @@ def model_should_run_on_event(model: str, event: str) -> bool:
     We put higher priority and fast models to pull request and rest to push.
     """
     if event == "pull_request":
-        return model in ["add", "ic3", "mv2", "mv3", "resnet18", "vit"]
+        return True or model in ["add", "ic3", "mv2", "mv3", "resnet18", "vit"]
```

Comment:
I will remove this once I get result.

Reply:
https://hud.pytorch.org/pytorch/executorch/commit/1f5993a750d58d3de6102337c871c98718c25b9b